x/auth/vesting/types/period.go

```diff
@@ -26,3 +27,267 @@ func (vp Periods) String() string {
 	return strings.TrimSpace(fmt.Sprintf(`Vesting Periods:
 		%s`, strings.Join(periodsListString, ", ")))
 }
+
+// A "schedule" is an increasing step function of Coins over time.
+// It's specified as an absolute start time and a sequence of relative
+// periods, with each step at the end of a period. A schedule may also
+// give the time and total value at the last step, which can speed
+// evaluation of the step function after the last step.
+
+// ReadSchedule returns the value of a schedule at readTime.
+func ReadSchedule(startTime, endTime int64, periods []Period, totalCoins sdk.Coins, readTime int64) sdk.Coins {
```

Comment:
```suggestion
// ReadSchedule returns the value of a schedule at the current provided time.
func ReadSchedule(startTime, endTime int64, periods []Period, totalCoins sdk.Coins, currTime int64) sdk.Coins {
```

IMO "readTime" is not that intuitive. I believe what's happening in this context, `readTime` should probably be called `currTime`, as in the current time at which the vesting account is evaluated in a tx execution.